extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -377,19 +377,14 @@ export class SchemaCompareDialog {
 		}
 
 		let values = cons.map(c => {
-			let db = c.options.databaseDisplayName;
 			let usr = c.options.user;
 			let srv = c.options.server;
 
-			if (!db) {
-				db = '<default>';
-			}
-
 			if (!usr) {
 				usr = 'default';
```

Comment:
doesn't this need to be localized?

Reply:
Yes. Also fixed it in the dacpac extension.